tools/ci/log4j.properties

```diff
@@ -37,15 +37,33 @@ appender.file.fileName = ${sys:log.dir}/mvn-${sys:mvn.forkNumber}.log
 appender.file.layout.type = PatternLayout
 appender.file.layout.pattern = %d{HH:mm:ss,SSS} [%20t] %-5p %-60c %x - %m%n
 
+# -----------------------------------------------------------------------------
+# ZooKeeper-related logs should end up here
+# -----------------------------------------------------------------------------
+appender.zk.name = ZooKeeperAppender
+appender.zk.type = FILE
+appender.zk.fileName = ${sys:log.dir}/zookeeper-${sys:mvn.forkNumber}.log
+appender.zk.layout.type = PatternLayout
+appender.zk.layout.pattern = %d{HH:mm:ss,SSS} [%20t] %-5p %-60c %x - %m%n
+
 # suppress the irrelevant (wrong) warnings from the netty channel handler
 logger.netty.name = org.jboss.netty.channel.DefaultChannelPipeline
 logger.netty.level = ERROR
+
+# collecting ZooKeeper-related logs in separate log file
 logger.zookeeper.name = org.apache.zookeeper
```

Comment:
Would this push both client & server logs into a new file? Wouldn't that be confusing to read?

Reply:
Isn't the client-side covered by curator? 🤔 Or what do you mean by client logs?

Reply:
I split the logs into client logs (`org.apache.flink.shaded.zookeeper3`) and server logs (`org.apache.zookeeper`)